src/components/CallTile.vue

```diff
@@ -199,19 +360,20 @@ export default {
     },
     // leave call, destroy call object, and reset local state values
     leaveAndCleanUp() {
+      if (!this.callObject) return;
       if (this.screen?.local) {
         this.callObject.stopScreenShare();
       }
       this.callObject.leave().then(() => {
-        this.callObject.destroy();
+        this.callObject?.destroy();
```

Comment:
I don't think you need this check, right?

Reply:
This is one of those 0.1% of the time edge cases. THEORETICALLY, someone could execute code like this:

```
this.callObject = null
```

In between the few hundred milliseconds that it takes `leave()` to resolve the `then` portion of the promise. This is why the linter is complaining if you don't add the `?`